spec/controllers/application_drafts_controller_spec.rb

```diff
@@ -233,6 +235,8 @@
           end
 
           it 'sends 1 mail to orga' do
+            pending "It changes by 3 instead of 1"
+            expect(enqueued_jobs.size).to eq 0
```

Comment:
Can we maybe fix this spec in this PR? Or isn't it valid any more?

Reply:
I added it to the todo list. TBH I am very unfamiliar with this corner of the app, and I'd rather not dive into it at the moment, because my head is already filled with all things Devise. 
Also, I hoped it would go away all by itself with PR #967 https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/967/files#diff-dc4cf03c7b58194272d6cea95a76d35e 😇

Reply:
Oh, is this failing on `master`? Sorry, I always assume that if a spec breaks on a PR it's because of something the PR changed (that's why I thought it would be related to this work).

Reply:
No you are totally right, it is failing in this PR 😊 , and not in master. I looked again, but I can't find why.
What do you think, can I leave it alone and see if it will be solved in #967 ?

Reply:
Ok, I looked into what `enqueued_jobs` had in it and it seems the 2 extra emails are confirmation emails for both students. As far as I understand the change in #967 is adding 2 new emails for the submitters (hence the change they did in the specs), so it's a different thing and won't fix the error we're seeing here (it will just say it expected the size to change by 3 but it changed by 5).

One way to solve this in the spec is to use `let!` when creating the draft, from what I tested.

Note: I'm a bit lost as to where in this PR we started sending the confirmation emails. Is it the change to `send_devise_notification`?

Reply:
Ooo that makes totally sense!!! We already sent confirmation emails, we just delivered them `now`, because the `deliver_later` implementation was only accessible for users created through github.  
Yes, that is the `send_devise_notification` moved to the user model in https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/1007/commits/5d2158b585e6176bece601eeaea0f230a18a1dec
I suspected it has to do with the create_users method, but couldn't find the `let!` solution. Thank you!!